README.md

```diff
@@ -9,7 +9,7 @@ your transactions are properly named and that your exceptions are properly logge
 
 ## Requirements
 
-- PHP >= 7.1
```

Comment:
Why change this?

Reply:
Only 7.2+ is tested on Travis and `composer.json` says:

https://github.com/digiaonline/lumen-newrelic/blob/80fe43e46454b7c3d88cc937e79734d223220035/composer.json#L12-L13

Both were changed in https://github.com/digiaonline/lumen-newrelic/pull/23.

Reply:
Oh, then it's fine!